public/app/features/explore/QueryRow.tsx

```diff
@@ -174,8 +176,11 @@ export class QueryRow extends PureComponent<QueryRowProps, QueryRowState> {
             </div>
           )}
           <div className="gf-form">
-            <button className="gf-form-label gf-form-label--btn" onClick={this.onClickClearButton}>
-              <i className="fa fa-times" />
+            <button
+              className="gf-form-label gf-form-label--btn"
+              onClick={() => this.onClickToggleHiddenQuery(this.props.query)}
+            >
+              <i className={`${!!this.props.query.hide ? 'fa fa-eye-slash' : 'fa fa-eye'}`} />
```

Comment:
No need to wrap in back-tick quotes here

Reply:
🤦‍♀️yes, not sure why I did this. Fixed as well.